site/content/docs/5.3/forms/validation.md

```diff
@@ -89,9 +89,8 @@ Custom feedback styles apply custom colors, borders, focus styles, and backgroun
   <div class="col-12">
     <div class="form-check">
       <input class="form-check-input" type="checkbox" value="" id="invalidCheck" required>
-      <label class="form-check-label" for="invalidCheck">
-        Agree to terms and conditions
-      </label>
+      <!-- Keep this <label> written on one single line to avoid browsers adding an extra and unwanted space between the end of the label ("conditions") and the asterisk meaning the field is required -->
+      <label class="form-check-label" for="invalidCheck">By checking this box, I agree to the <a href="" target="_blank" aria-label="Terms and conditions (opens new window)"> terms and conditions</a></label>
```

Comment:
In all the occurrences in this PR, why is there an extra space at the beginning of `> terms and conditions</a></label>`? Should be removed IMO.